samples/typedefs/src/main/resources/DataModel.pkl

```diff
@@ -206,6 +206,14 @@ customTypes {
         description = "Subject area of the entity."
         type = "string"
       }
+      ["\(a)\(dmEntity)GeneralizationName"] {
+        description = "Name of the general entity."
+        type = "string"
+      }
+      ["\(a)\(dmEntity)GeneralizationQualifiedName"] {
+        description = "Unique identifier for the general entity."
+        type = "string"
+      }
```

Comment:
Have a look at the supertype-level denormalized attributes (e.g. line 70+) to see which indexing to use for the normal name vs qualifiedName — also, I think there are variables defined you can use for the `Name` and `QualifiedName` portion (just to avoid any potential typos)